maestro-cli/src/main/java/maestro/cli/api/ApiClient.kt

```diff
@@ -103,6 +104,7 @@ class ApiClient(
             .setType(MultipartBody.FORM)
             .addFormDataPart("app_binary", "app.zip", appFile.toFile().asRequestBody("application/zip".toMediaType()).observable(progressListener))
             .addFormDataPart("workspace", "workspace.zip", workspaceZip.toFile().asRequestBody("application/zip".toMediaType()))
+            .addFormDataPart("mapping", "mapping.txt", workspaceZip.toFile().asRequestBody("text/plain".toMediaType()))
```

Comment:
@artem888 The dsym of iOS are upload with same key right?

Reply:
Yes I believe so

Reply:
yes , it is